drivers/axi_core/jesd204/axi_adxcvr.c

```diff
@@ -232,8 +233,8 @@ int adxcvr_clk_set_rate(struct adxcvr *xcvr,
 	uint32_t i;
 	int ret;
 
-//	dev_dbg(st->dev, "%s: Rate %lu Hz Parent Rate %lu Hz",
-//			__func__, rate, parent_rate);
+	pr_debug("%s: Rate %lu Hz Parent Rate %lu Hz\n",
```

Comment:
it'd do this directly in the first commit, instead of commenting code and changing it afterwards. check the entire commit for similar changes.

Reply:
This was done so that specific modifications are separated in different commits. Squashing the commits would turn the PR into a large commit with a large amount of different modifications.